BaseClasses.py

```diff
@@ -546,7 +546,7 @@ def location_relevant(location: Location):
         def all_done():
             """Check if all access rules are fulfilled"""
             if beatable_fulfilled:
-                if any(location_conditition(location) for location in locations):
+                if any(location_condition(location) for location in locations):
```

Comment:
I intended this for a different branch I was working on, not sure how it ended up in here, but I'm gonna leave it.